pom.xml

```diff
@@ -887,6 +890,27 @@
         </plugins>
     </build>
     
+    <profiles>
+        <profile>
+            <id>jdk11+</id>
+            <activation>
+                <jdk>[11,)</jdk>
```

Comment:
The javax annotation was removed since JDK 9. So maybe the low bound should be 1.9?

Reply:
Since JDK 9 is not a LTS verion, I think it will not be in the supported JDK list.